docs/datastorage/saveddata.md

```diff
@@ -11,16 +11,16 @@ Declaration
 Each SD implementation must subtype the `SavedData` class. There are two important methods to be aware of:
 
 * `save`: Allows the implementation to write NBT data to the level.
-* `setDirty`: A method that must be called after changing the data, to notify the game that there are changes that need to be written. If not called, `#save` will not get called and the existing data will persist.
+* `setDirty`: A method that must be called after changing the data, to notify the game that there are changes that need to be written. If not called, `#save` will not get called and the original data will remain unchanged.
 
 Attaching to a Level
 ----------------------
```

Comment:
While you're at it, could you migrate this file to the new header standard? (# h1 / ## h2 instead of == and --)